core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -179,6 +180,7 @@
 
   @Test void testPushDownSort() {
     CalciteAssert.model(JdbcTest.SCOTT_MODEL)
+        .with(CalciteConnectionProperty.TOPDOWN_OPT.camelName(), false)
```

Comment:
why do we need this?

Reply:
In this case, the new planner is producing a plan with exactly the same cost. 
However, the new plan will affect later tests' assertion. I didn't want to modified so many of them. So I added this line instead of modifying the expects.